<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>829</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="centraWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>811</width>
     <height>516</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(246, 245, 244);</string>
   </property>
   <widget class="QWidget" name="topWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>811</width>
      <height>96</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(72, 73, 77);
border-top-left-radius: 10px;
border-top-right-radius: 10px;</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="2" rowspan="2">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>258</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="curUserLab">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
font: 700 italic 11pt &quot;Yrsa&quot;;</string>
       </property>
       <property name="text">
        <string>not log in</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>268</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="1" rowspan="2">
      <widget class="QLabel" name="curChatLab">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
font: 700 italic 11pt &quot;Yrsa&quot;;</string>
       </property>
       <property name="text">
        <string>chating</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2">
      <widget class="QWidget" name="widget" native="true">
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>0</y>
          <width>31</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>x</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="chatWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>96</y>
      <width>575</width>
      <height>358</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QListWidget" name="chatMsgListWidget"/>
     </item>
     <item row="1" column="0">
      <widget class="QTextEdit" name="chatMsgSendEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="rightWidget" native="true">
    <property name="geometry">
     <rect>
      <x>581</x>
      <y>96</y>
      <width>230</width>
      <height>420</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QWidget" name="friendList" native="true">
       <widget class="QWidget" name="friendTop" native="true">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>211</width>
          <height>41</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>124</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="addFriend">
           <property name="styleSheet">
            <string notr="true">QPushButton {
	width: 60px;
	height: 30px;
	border: none;
	background-color: rgb(20, 180, 206);
	color: rgb(255, 255, 255);
	font: 700 italic 11pt &quot;Yrsa&quot;;
	border-radius: 5px;
}
</string>
           </property>
           <property name="text">
            <string>addFriend</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QListWidget" name="friendMess">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>201</width>
          <height>151</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="groupList" native="true">
       <widget class="QWidget" name="groupTop" native="true">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>201</width>
          <height>41</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>114</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="addGroup">
           <property name="styleSheet">
            <string notr="true">QPushButton {
	width: 60px;
	height: 30px;
	border: none;
	background-color: rgb(20, 180, 206);
	color: rgb(255, 255, 255);
	font: 700 italic 11pt &quot;Yrsa&quot;;
	border-radius: 5px;
}</string>
           </property>
           <property name="text">
            <string>addGroup</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QListWidget" name="groupMess">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>201</width>
          <height>151</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>460</y>
      <width>575</width>
      <height>43</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>@YXZ</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>467</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="Send">
       <property name="styleSheet">
        <string notr="true">QPushButton {
	width: 60px;
	height: 30px;
	border: none;
	background-color: rgb(20, 180, 206);
	color: rgb(255, 255, 255);
	font: 700 italic 11pt &quot;Yrsa&quot;;
	 border-radius: 5px;
}</string>
       </property>
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
